Source/ui/EditToneTab.cpp

```diff
@@ -1098,12 +1098,14 @@ void EditToneTab::comboBoxChanged(juce::ComboBox* button)
 void EditToneTab::sendSysexPatchToneChange1Byte(uint8_t address, uint8_t value)
 {
     uint8_t data[5];
-    data[0] = 0x00; // address MSB
-    data[1] = 0x08; // address
+    data[0] = 0x00;              // address MSB
+    data[1] = 0x08;              // address
     data[2] = 0x28 + toneCount;  // address
-    data[3] = address & 127;  // address LSB
-    data[4] = value;                 // data
+    data[3] = address & 127;     // address LSB
+    data[4] = value;             // data
+
     uint32_t checksum = 0;
+
     for (size_t i = 0; i < 5; i++) {

```

Comment:
Do we have a bunch of different slightly different implementations of checksum computation? we'll need to clean this up eventually I think